client/cert_authority_site_create.go

```diff
@@ -0,0 +1,20 @@
+package client
+
+import (
+	"github.com/skupperproject/skupper/api/types"
+	"github.com/skupperproject/skupper/pkg/kube"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (cli *VanClient) CASiteCreate(ownerRef *metav1.OwnerReference) error {
```

Comment:
Again, minor issue on naming, but I think the purpose of the method would be clearer as ServiceCACreate or similar.

Reply:
Sure! makes more sense

Reply:
Done!